checkpoint_engine/ps.py

```diff
@@ -889,7 +952,7 @@ def gather_metas(self, checkpoint_name: str):
                     ptr=x.buffer.data_ptr(),
                     size=x.size,
                 )
-                for x in self._memory_pool.get(checkpoint_name, [])
+                for x in (self._get_memory_pool(checkpoint_name) or [])
```

Comment:
Here may cause RuntimeError if checkpoint_name does not exist in self._memory_pool